docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
```

Comment:
Do we want to point to the [Prebuilt rule reference](https://www.elastic.co/guide/en/security/8.8/prebuilt-rules.html) page? AFIAK, the page isn't maintained anymore and it's last update was in 8.6. Would it make more sense to link to the [Downloadable rule updates](https://www.elastic.co/guide/en/security/master/prebuilt-rules-downloadable-updates.html) page?

cc @terrancedejesus since we [talked](https://elastic.slack.com/archives/CED7QCL3E/p1684435236800429?thread_ts=1684419596.465149&cid=CED7QCL3E) about this page's maintenance a couple of months ago.

Reply:
👋🏽 @nastasha-solomon 

We actually were asked by @approksiu and AET to update our process to update this doc every release. We are tracking this request and work here -> https://github.com/elastic/detection-rules/issues/2867